containers/src/Kokkos_DualView.hpp

```diff
@@ -209,11 +209,6 @@ class DualView : public ViewTraits<DataType, Properties...> {
   t_modified_flags modified_flags;
 
  public:
-  // does the DualView only reference one View
-  static constexpr bool impl_dualview_stores_single_view =
-      SpaceAccessibility<Kokkos::HostSpace,
-                         typename t_dev::memory_space>::accessible;
-
```

Comment:
Just keep this replacing `impl_dualview_stores_single_view` with `impl_dualview_is_single_device` and
```
value = std::is_same_v<typename t_dev::device_type,
                             typename t_host::device_type>;
```